backend/controller/contentController.js

```diff
@@ -268,22 +270,25 @@ const httpDeleteById = asyncHandler(async (req, res, next) => {
 
   const { thumbnail, trailer, content } = contentData;
 
-  // FIXME: Fix this as discussed (MANG)
-  // perform delete in cloudinary
   if (contentData) {
```

Comment:
if (contentData)? Why this? contentData looks like an object...

Reply:
this condition check is removed

Reply:
You have removed this, are you checking the thumbnail, tailter, content then?